### PR TITLE
Add more descriptive features into setup.py for pip and pypi.

### PR DESCRIPTION
**Pull request by loechel**
Project URLs to show in PyPI https://packaging.python.org/guides/distributing-packages-using-setuptools/#project-urls

and necessary Python Version Check on Install : https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires From PEP 440 https://www.python.org/dev/peps/pep-0440/

### CODE REVIEW

**Comment by loechel** [comment]
ok, appveyor did install an incompatile pytest-html which has f-strings in it on Python 2.7 and 3.5 that is not ok, but nothing related to that pull request

File: setup.py [hunk 1]
```diff
@@ -60,11 +61,17 @@ def read(*rnames):
...
+        "Documentation": "https://restrictedpython.readthedocs.io/",
+        "Source": "https://github.com/zopefoundation/RestrictedPython",
+        "Tracker": "https://github.com/zopefoundation/RestrictedPython/issues",
+    },
     packages=find_packages('src'),
     package_dir={'': 'src'},
     install_requires=[
         'setuptools',
     ],
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4 ",
```
Comment (icemac) [inline setup.py]:
```suggestion
    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9 ",
```

This way we make sure it does not run with an untested Python version.

Comment (loechel) [reply setup.py]: This would mean for us, that we need to adjust or work on special branches for each new Python Version, but ok, I will change

**Comment by icemac** [comment]
See #183 which should fix the tests.

**Comment by icemac** [comment]
Coveralls drives me crazy. Maybe we should look for another coverage service.

**Pull request merged**